components/oncehub/package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "@pipedream/oncehub",
-  "version": "0.0.3",
+  "version": "0.0.4",
```

Comment:
I think this should be `0.1.0` [since it introduces new components](https://pipedream.com/docs/components/guidelines/#versioning)